src/main/java/io/chef/jenkins/ChefIdentity.java

```diff
@@ -36,23 +36,40 @@
  * @author tfitch
  */
 public class ChefIdentity implements Serializable {
+	public static final String SHELL_SCRIPT_FILE = "install-chef-cookbook.sh";
+	public static final String REMOTE_HOST_CLIENT_RB_FILE = "remote-host-client.rb.conf";
```

Comment:
It looks like these two lines imply a user will have the files `install-chef-cookbook.sh` and `remote-host-client.rb.conf` in their projects, but the plugin is not going to create them for them, right?

I don't like this because it makes the plugin too fragile.  It should create empty versions of these files, or have examples of these files for users to copy from.

Or do you have that and I missed it?